features/steps/partcad-cli/commands/version.py

```
@@ -29,6 +50,15 @@ def step_impl(context, substring):
     assert substring in strip_ansi(context.result.stderr)
 
 
+@then('STDOUT should not contain "{substring}" with path')
+def step_impl(context, substring):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 45

(F811)